### DISCUSSION TITLE
No population-specific enriched motifs in marker peaks

### DISCUSSION THREAD

**Discussion by mitchTwoTimes** [GreenleafLab/ArchR#1761] [Questions / Documentation]
Hi,

Wondering if the maintainer or anyone else has tips for improving the results of enriched motifs in marker peaks ([Sect 12.2](https://www.archrproject.com/bookdown/motif-enrichment-in-marker-peaks.html)). I've used both the default cisbp `motifSet`, as well as a custom ChIP-Seq annotation. I've pasted an example of what I'm consistently seeing, where the last two populations get little to no enrichment.

My thresholds in `cutoff` are already pretty liberal:
```
markersPeaks <- getMarkerFeatures(
  ArchRProj = proj, 
  useMatrix = "PeakMatrix", 
  groupBy = "Clusters_atac_all",
  bias = c("TSSEnrichment", "log10(nFrags)"),
  testMethod = "wilcoxon"
)

enrichMotifs <- peakAnnoEnrichment(
  seMarker = markersPeaks,
  ArchRProj = proj,
  peakAnnotation = "TF_binding", # Motif
  cutOff = "FDR <= 0.1 & Log2FC >= 0.5"
)

```

Would increasing the resolution of my clusters help? Should I tweak parameters in `getMarkerFeatures`? Or is there a different interpretation I am missing?

![image](https://user-images.githubusercontent.com/92815743/205193108-6d4aed41-92a8-4b18-841e-1a80802a25b1.png)

**Comment by rcorces** [comment]
Its a normalized enrichment, so I dont think its that they get no enrichment, just that their enrichment is tiny compared to your other clusters. You see the same thing in GMP cells in the tutorial:
https://www.archrproject.com/bookdown/motif-enrichment-in-marker-peaks.html
I think this is probably related to both biological and technical variables in the experiment.
